js/components/SpidLoginButton.js

```diff
@@ -178,7 +180,8 @@ class IdpSelectionScreen extends React.Component {
     closeModal: () => void,
     onSelectIdp: IdentityProvider => void,
     onSpidLogin: (string, string) => void,
-    onSpidLoginError: string => void
+    onSpidLoginError: string => void,
+    userState: any
```

Comment:
do not use `any`, we do have a type for [UserState](https://github.com/teamdigitale/italia-app/blob/master/js/reducers/user.js#L37)